src/components/EventTheme/colors.tsx

```diff
@@ -0,0 +1,76 @@
+import { EventName } from "~/data/types";
+
+interface EventColors {
```

Comment:
[Suggestion] Since this interface is used in things available outside of this file (the type of `colors`), you might want to export it. It doesn't make a difference now but later on someone else might want to refer to it, and it can be confusing not having the interface available.

```suggestion
export interface EventColors {
```

(non-blocking, just a thought)